Client/multiplayer_sa/CMultiplayerSA.cpp

```diff
@@ -1486,6 +1486,12 @@ void CMultiplayerSA::InitHooks()
     MemSet((void*)0x72925D, 0x1, 1);            // objects
     MemSet((void*)0x729263, 0x1, 1);            // players
 
+    
+    // Allow crouching with 1HP
+    float fDuckingHealthThreshold;
+    MemPut((void*)0x6943AD, &fDuckingHealthThreshold);
```

Comment:
`&fDuckingHealthThreshold` will be a memory location on the stack. Is that intended?

Reply:
i had to do it in that way due it refer to global float number
0x6943AB: `D8 1D 24 86 85 00                       fcomp   ds:fOne         ; 1.0f`
and that variable is used in 2855 other locations what cause i just can't do simple `memset(0x6943AD, 0)`

Reply:
`float fDuckingHealthThreshold` should not be a local variable

Reply:
is it now correct?

Reply:
Yes